crates/bevy_ui/src/ui_node.rs

```diff
@@ -53,29 +60,54 @@ impl AddAssign<f32> for Val {
     }
 }
 
+/// Describes the style of a UI node, these track with the FlexBox system, see <https://cssreference.io/flexbox/>
+/// Note: Bevy's UI is upside down compared to how FlexBox normally works, to stay consistent with engine paradigms about layouting from
+/// the upper left corner of the display
 #[derive(Component, Clone, PartialEq, Debug, Reflect)]
 #[reflect(Component, PartialEq)]
 pub struct Style {
+    /// Whether to arrange this node and its children with flexbox layout
     pub display: Display,
+    /// Whether to arrange this node relative to other nodes, or positioned absolutely
     pub position_type: PositionType,
+    /// Which direction the content of this node should go
     pub direction: Direction,
+    /// Whether to use column or row layout
     pub flex_direction: FlexDirection,
+    /// How to wrap nodes
     pub flex_wrap: FlexWrap,
+    /// How items are aligned according to the cross axis
     pub align_items: AlignItems,
+    /// Like align_items but for only this item
     pub align_self: AlignSelf,
+    /// How to align each line, only applies if flex_wrap is set to
+    /// [`FlexWrap::Wrap`] and there are multiple lines of items
     pub align_content: AlignContent,
+    /// How items align according to the main axis
     pub justify_content: JustifyContent,
+    /// The position of the node as descrided by its Rect
     pub position: Rect<Val>,
+    /// The margin of the node
     pub margin: Rect<Val>,
+    /// The padding of the node
     pub padding: Rect<Val>,
+    /// The border of the node
     pub border: Rect<Val>,
+    /// Shorthand property for flex_direction and flex_wrap
```

Comment:
This seems incorrect

Reply:
Yep, accidentally input the description for flex-flow instead